xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/jackson/modules/MpTokenIssuanceIdDeserializer.java

```diff
@@ -0,0 +1,46 @@
+package org.xrpl.xrpl4j.model.jackson.modules;
+
+/*-
+ * ========================LICENSE_START=================================
+ * xrpl4j :: model
+ * %%
+ * Copyright (C) 2020 - 2022 XRPL Foundation and its contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import org.xrpl.xrpl4j.model.transactions.MpTokenIssuanceId;
+
+import java.io.IOException;
+
+/**
+ * Custom Jackson deserializer for {@link MpTokenIssuanceId}s.
+ */
+public class MpTokenIssuanceIdDeserializer extends StdDeserializer<MpTokenIssuanceId> {
+
+  /**
+   * No-args constructor.
+   */
+  public MpTokenIssuanceIdDeserializer() {
+    super(MpTokenIssuanceId.class);
+  }
+
+  @Override
+  public MpTokenIssuanceId deserialize(JsonParser jsonParser, DeserializationContext ctxt) throws IOException {
+    return MpTokenIssuanceId.of(jsonParser.getText());
```

Comment:
Is this value expected to be in HEX? Just thinking out loud if `MpTokenIssuanceId` should be represented as a `String` in Java.

Reply:
I believe so? Isn't it the hash of the issuer address concatenated with the sequence?